### DISCUSSION TITLE
Docker

### DISCUSSION THREAD

**Discussion by bhaktasn** [ICube-Robotics/ethercat_driver_ros2#105] [Q&A]
Anyone have any success running the Docker container? Could use a pointer. Looking to run 
```bash
/etc/init.d/ethercat start
ethercat slaves
```
as a test

**Comment by yguel** [comment]
Just to close this discussion. As ethercat master IgH for 1.5 works as a kernel module, it cannot work correctly in a docker container. So this kind of test inside a docker container for a 1.5 version of IgH master will not work.
